nms/app/packages/magmalte/app/views/subscriber/SubscriberDetail.js

```diff
@@ -158,6 +163,37 @@ export default function SubscriberDetail() {
     </>
   );
 }
+function StatusInfo() {
+  const {match} = useRouter();
+  const subscriberId: string = nullthrows(match.params.subscriberId);
+  const networkId: string = nullthrows(match.params.networkId);
+  const enqueueSnackbar = useEnqueueSnackbar();
+
+  const ctx = useRefreshingContext({
+    context: SubscriberContext,
+    networkId: networkId,
+    type: 'subscriber',
+    interval: REFRESH_INTERVAL,
+    enqueueSnackbar: enqueueSnackbar,
```

Comment:
in case of an error, let's skip printing every 30 seconds. Retain the attribute in the prop, but you can remove it here.